src/seabirdscientific/cal_coefficients.py

```diff
@@ -185,3 +185,19 @@ def __init__(self, im, a0, a1, multiplier):
         self.a0 = a0
         self.a1 = a1
         self.multiplier = multiplier
+
+
+class PHSeaFETCoefficients:
+    """_summary_
+    """
+    def __init__(self, int_k0, int_k2, ext_k0, ext_k2, f1, f2, f3, f4, f5, f6):
+        self.int_k0 = int_k0
+        self.int_k2 = int_k2
+        self.ext_k0 = ext_k0
+        self.ext_k2 = ext_k2
+        self.f1 = f1
+        self.f2 = f2
+        self.f3 = f3
+        self.f4 = f4
+        self.f5 = f5
+        self.f6 = f6
```

Comment:
I think we should have a separate class for external and internal. The deep SeapHox doesn't have an internal pH sensor, so the user would have to put in filler values for int_k0 and int_k2. Similarly the f coefficients aren't used for SeaFETs (always shallow version).